.github/workflows/publish-commit-bottles.yml

```diff
@@ -25,8 +25,7 @@ permissions:
 
 jobs:
   upload:
-    runs-on: ${{github.event.inputs.self_hosted == 'true' && 'linux-self-hosted-1' || 'ubuntu-22.04'}}
-    container: ${{github.event.inputs.self_hosted == 'true' && fromJson('{"image":"ghcr.io/homebrew/ubuntu22.04:master","options":"--user=linuxbrew -e GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED"}') || ''}}
+    runs-on: ${{github.event.inputs.large_runner == 'true' && 'homebrew-large-bottle-upload' || 'ubuntu-22.04'}}
```

Comment:
I think `github.event.inputs.large_runner` will get cast into a boolean anyway, so comparing against `'true'` is probably not needed.

Not sure if this is worth changing here.